### PR TITLE
Generate additional fields in text models db.json

### PR DESCRIPTION
**Pull request by ceruleandeep**
This is a proof-of-concept in relation to https://github.com/Haidra-Org/AI-Horde/issues/452

### CODE REVIEW

**Comment by db0** [comment]
Personally I would have preferred you switch to just maintaining the json directly, but I understand that's a bit of a PITA when you have to make 3 copies per model. This part should be necessary much longer however.

btw, if you need to add more fields to the json, that's also allowed. You don't have to stick with the existing ones only.

**Comment by ceruleandeep** [comment]
For this PR I've tried to:

- not add to @candre23 's maintenance burden or change the workflow (can still whitelist models by adding 2 fields and running `convert`)
- avoid breaking changes to db.json (no fields removed, no semantics changed, additions only) 
- populate a decent amount of example data into db.json so you can see what I'm trying to get from an API endpoint and you have something to work with

Basically trying to keep this bit small and isolated because https://github.com/Haidra-Org/AI-Horde/issues/452 probably has several moving parts already :)
